sh.sh

```diff
@@ -0,0 +1,32 @@
+read _ _ KERNEL _ </proc/version
+
+[ -z "$HOSTNAME" ] && read HOSTNAME </etc/hostname
```

Comment:
don't need to test for hostname, testing is ignored everywhere. go/rust would be twice as long if we did test.